doc/developer/design/20230126_designs_reviews.md

```diff
@@ -0,0 +1,99 @@
+# Scaling our design process
+
+## Summary
+
+<!--
+// Brief, high-level overview. A few sentences long.
+// Be sure to capture the customer impact - framing this as a release note may be useful.
+-->
+
+The current design process we're using at Materialize shows its limitations.
+Engineers write designs in Notion because it's easy to start, but we lack any established review process, and following discussions is difficult.
+Writing designs as pull-requests is perceived to be more tedious than simply writing the content in Notion.
+To address this, we analyze the current shortcomings and propose updates to the existing process.
+
+## Goals
+
+<!--
+// Enumerate the concrete goals that are in scope for the project.
+-->
+
+We aim at creating a design process that is lightweight, captures reviews and specific snapshots in time, is open, while providing a clear benefit to engineers.
+Let's break down these aspects.
+
+A design review process needs to empower engineers to focus on the task at hands.
+It helps to talk about goals, how they fit into the product roadmap, and outline solutions.
+An engineer can present different solutions, and it's expected that new alternatives come up while working on a design.
+Explaining trade-offs helps to motivate a specific solution.
+
+Creating a design is an iterative process.
+Capturing discussions is an important aspect to understand how an opinion was formed and how designs changed during the review.
+We need to capture both, and Notion doesn't give us a good way to capture discussions and specific versions.
+
+Materialize defaults to leading discussions in the open, unless it's not permissible.
+This ensures that we uphold a high standard, and it gives engineers outside visibility into their work.
```

Comment:
> and it gives engineers outside visibility into their work.

I'm not sure how to parse this sentence. Is it "gives engineers (outside visibility) into their work" or "gives (engineers outside) visibility into their work"? In the first case I'd wonder what "outside visibility" is, in the second case I'd wonder who "their" is.